### DISCUSSION TITLE
Bing questions - Is there any way to tell Bing to use creative mode? Any way to process longer text

### DISCUSSION THREAD

**Discussion by MalcolmHfx** [xtekky/gpt4free#1311] [General]
response = g4f.ChatCompletion.create(
                model="gpt-4",
                #model="gpt-3.5-turbo",
                provider=g4f.Provider.Bing,
                messages=[{"role": "user", "content": preamble_text + extracted_text}],
                stream=True,
            )    

Also, I am getting this response sometimes, presumably when I send too much input. Any tips here?

I'm sorry, but I cannot summarize this discussion for you. It is too long and complex for me to process in a single turn. Please try to provide a shorter and simpler input for me to work with. 😅

**Comment by hlohaus** [comment]
You can pass `tone="creative"` for creative mode. For your long input i don't have a solution.

**Reply by zeke789** [reply]
hey, that's not working !  Look:

response = g4f.ChatCompletion.create(
    model="gpt-4",
    messages=[{"role": "user", "content": "hola, en que modo estas mostrandome las respuestas? creativo,equilibrado o preciso??"}],
    stream=True,
    tone="Creative"
)
for message in response:
    print(message, flush=True, end='')
    
    
And always this is the response  from bing when i ask what tone is using:
   "Hello, this is Bing. I am in **balanced** mode, that means..... "
